### PR TITLE
Feature/inversion details

### PR DESCRIPTION
**Pull request by benjamineac**
Inversion Mdf tab shows meta data, test suite set up.

### CODE REVIEW

File: src/components/InversionSolutionMfdTab.tsx [hunk 1]
```diff
@@ -75,7 +75,12 @@ const InversionSolutionMfdTab: React.FC<InversionSolutionMfdTabProps> = ({
     maxMagnitude = 9.0;
     minMagnitude = 5.0;
   }
-
+  //Removes filename from inversion data
+  const cleanedMeta = data?.node?.meta?.filter((el) => {
+    return el?.k !== 'rupture_set';
```
Comment (chrisbc) [inline src/components/InversionSolutionMfdTab.tsx:80]: @benjamineac can you also remove 'rupture_set_file_id' as it's not useful context here.

**Review by chrisbc — CHANGES_REQUESTED** [review]
pls see comment to remove one more kv pair, otherwise cool thanks!.

**Review by chrisbc — COMMENTED** [review]
@benjamineac I've switched target branch to deploy-test. Pls don't choose main as your PR target. I only just caught that!

File: src/components/InversionSolutionMfdTab.tsx (hunk 1 above)
Comment (benjamineac) [reply src/components/InversionSolutionMfdTab.tsx:80]: Yep!

**Review by chrisbc — APPROVED** [review]
Thanks!

**Pull request merged**